framework/db/oci/Schema.php

```diff
@@ -523,4 +525,25 @@ public function insert($table, $columns)
 
         return $result;
     }
+
+    /**
+     * Sets the isolation level of the current transaction.
+     * @param string $level The transaction isolation level to use for this transaction.
+     * This can be either [[Transaction::READ_COMMITTED]] or [[Transaction::SERIALIZABLE]].
+     * @throws \yii\base\NotSupportedException when unsupported isolation levels are used.
+     * Oracle only supports SERIALIZABLE and READ COMMITTED.
+     * @see http://docs.oracle.com/cd/E11882_01/server.112/e41084/statements_10005.htm#SQLRF01705
+     * @since 2.0.12
```

Comment:
`@since` is only needed if the interface changes. this method existed before 2.0.12 in the parent class so it is not new.